resources/views/listing/partials/item.blade.php

```diff
@@ -16,7 +16,7 @@ class="mb-3 h-48 w-full rounded-t object-contain" :alt="item.name" :loading="con
                 />
                 <x-rapidez::no-image v-else class="mb-3 h-48 rounded-t" />
                 <div class="px-2">
-                    <div class="text-base font-medium text-gray-900">@{{ item.name }}</div>
+                    <div class="text-base font-medium text">@{{ item.name }}</div>
```

Comment:
`text` is the default, do we really need it here? The same for `text-base` right?